### PR TITLE
use plugin configuration

### PR DESCRIPTION
**Pull request by benjijamorris**
## What does this PR do?

Incorporate plugin's bump workflow

## Before submitting

- [x] Did you make sure **title is self-explanatory** and **the description concisely explains the PR**?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you list all the **breaking changes** introduced by this pull request?
- [x] Did you **test your PR locally** with `pytest` command?
- [x] Did you **run pre-commit hooks** with `pre-commit run -a` command?

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

File: scripts/publish_bumpver_handler.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  - expect: ValueError
+
+- python publish_bumpver_handler.py patch
+  - expect: version updated to 2.1.2
+
+- git reset --hard {hash of the commit made at the beginning}
+- git tag --delete 1.0.0 2.0.0 2.1.0 2.1.1 2.1.2 2.1.2.dev0 2.1.2.dev1
+"""
+
+import subprocess  # nosec
```
Comment (saeliddp) [inline scripts/publish_bumpver_handler.py:44]: What does `#nosec` do?

File: version.toml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# -----DO NOT MODIFY THIS FILE-----
+# This file should only be modified by bumpver, which should in turn only be ran
+# via a GH Action
+version = "0.1.7"
```
Comment (saeliddp) [inline version.toml:4]: I think in order to complete this setup, you need to have something like [this](https://github.com/AllenCell/allencell-ml-segmenter/blob/main/.github/workflows/publish.yaml) which watches for modifications to `version.toml` then builds and publishes if `version.toml` has been modified. You could also build and publish manually if you prefer

Comment (saeliddp) [reply version.toml:4]: Note that ours is currently set up to publish to artifactory, so yours might look more like [this](https://github.com/AllenCell/allencell-ml-segmenter/commit/a5cd3d7deb3dc200b0e0c8e79b7c2a936b3ec5c5)

File: scripts/publish_bumpver_handler.py (hunk 1 above)
Comment (benjijamorris) [reply scripts/publish_bumpver_handler.py:44]: bandit (part of our precommit) flags possible security vulnerabilities and it was mad about subprocess use, nosec skips that line

**Review by hughes036 — DISMISSED** [review]

File: version.toml (hunk 2 above)
Comment (benjijamorris) [reply version.toml:4]: ah good point, I'll modify our publish.yml

File: .github/workflows/publish.yml
Comment (benjijamorris) [inline .github/workflows/publish.yml:1]: @saeliddp  updated here

**Review by saeliddp — APPROVED** [review]

**Pull request merged**